packages/ember-intl-lint/package.json

```diff
@@ -14,7 +14,8 @@
   "homepage": "https://ember-intl.github.io/ember-intl/",
   "repository": {
     "type": "git",
-    "url": "https://github.com/ember-intl/ember-intl.git"
+    "url": "https://github.com/ember-intl/ember-intl.git",
+    "directory": "packages/ember-intl-lint"
```

Comment:
Currently, redirections (e.g. to `CONTRIBUTING`) from the `npm` page don't work. Going to try out `repository.directory` for this.

https://docs.npmjs.com/cli/v11/configuring-npm/package-json#repository

> If the `package.json` for your package is not in the root directory (for example if it is part of a monorepo), you can specify the directory in which it lives:
>
> ```json
> {
>   "repository": {
>     "type": "git",
>     "url": "git+https://github.com/npm/cli.git",
>     "directory": "workspaces/libnpmpublish"
>   }
> }
> ```